python_analysis_env.yml

```diff
@@ -18,4 +18,6 @@ dependencies:
   - conda-forge::umap-learn
   - conda-forge::matplotlib
   - pip:
-    - git+https://github.com/cytomining/pycytominer@afac3ea16818ad25f37318ecd5c5090c0eff5806
+    - pycytominer
```

Comment:
Consider using conda-forge to download Pycytominer unless there's a reason to use `pip`:
https://anaconda.org/conda-forge/pycytominer

Reply:
I will keep it as pip for now since all other preprocessing software were installed this way. I will change in further projects, thanks for pointing it out!